src/java.base/share/classes/java/lang/runtime/SwitchBootstraps.java

```diff
@@ -235,13 +245,51 @@ public static CallSite enumSwitch(MethodHandles.Lookup lookup,
         labels = labels.clone();
 
         Class<?> enumClass = invocationType.parameterType(0);
+        Stream.of(labels).forEach(l -> validateEnumLabel(enumClass, l));
+        MutableCallSite result =
+                new MutableCallSite(invocationType);
+
+        MethodHandle temporary =
+                MethodHandles.insertArguments(LAZY_DO_ENUM_SWITCH, 2, labels, lookup, enumClass, result);
+        temporary = temporary.asType(invocationType);
+
+        result.setTarget(temporary);
+
+        return result;
+    }
+
+    private static <E extends Enum<E>> void validateEnumLabel(Class<?> enumClassTemplate, Object label) {
+        if (label == null) {
+            throw new IllegalArgumentException("null label found");
```

Comment:
I think that this one is not hit by any test

Reply:
Should be resolved by:
https://github.com/openjdk/jdk/pull/13074/commits/bb59254e409821c9b14287c73a35ebe2dd2cbda9

Thanks for the comment!